packages/lesswrong/components/recentDiscussion/RecentDiscussionTag.tsx

```diff
@@ -146,6 +187,7 @@ const RecentDiscussionTag = ({ tag, refetch = () => {}, comments, expandAllThrea
               comment={comment.item}
               childComments={comment.children}
               key={comment.item._id}
+              showParentDefault
```

Comment:
This will show the parent comment by default, to provide more context. In a future PR, we'll actually display the root comment text as well, similar to how the post body preview looks.